recipes/vegan_brownies.qmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Vegan Brownies"
+image: "https://img.taste.com.au/5hsR3Ore/w720-h480-cfill-q80/taste/2016/11/classic-chewy-brownie-102727-1.jpeg"
+image-alt: "https://img.taste.com.au/5hsR3Ore/w720-h480-cfill-q80/taste/2016/11/classic-chewy-brownie-102727-1.jpeg"
+categories:
+  - Vegan
```

Comment:
Please add your author information to the YAML header:

```yaml
author: <your name here>
```

Reply:
I added my name to the Quarto file.